src/OrbitQt/CallTreeViewItemModel.cpp

```diff
@@ -161,10 +196,23 @@ QVariant CallTreeViewItemModel::GetForegroundRoleData(const QModelIndex& index)
   ORBIT_CHECK(index.isValid());
   auto* item = static_cast<CallTreeNode*>(index.internalPointer());
   auto* unwind_errors_item = dynamic_cast<CallTreeUnwindErrors*>(item);
-  if (unwind_errors_item != nullptr && index.column() == kThreadOrFunction) {
-    static const QColor kUnwindErrorsColor{Qt::red};
+  auto* unwind_error_item = dynamic_cast<CallTreeUnwindErrorType*>(item);
+
+  if ((unwind_errors_item != nullptr || unwind_error_item != nullptr) &&
+      index.column() == kThreadOrFunction) {
+    static const QColor kUnwindErrorsColor{QColor::fromRgb(255, 128, 0)};
     return kUnwindErrorsColor;
   }
+
+  const auto* parent_is_unwind_errors_item =
+      dynamic_cast<const CallTreeUnwindErrors*>(item->parent());
+  const auto* parent_is_unwind_error_item =
+      dynamic_cast<const CallTreeUnwindErrorType*>(item->parent());
+  if ((parent_is_unwind_error_item != nullptr || parent_is_unwind_errors_item != nullptr) &&
```

Comment:
I think this is fine but now `parent_is_unwind_errors_item != nullptr` is always false, isn't it?

Reply:
Good catch. I forgot to remove this.